.github/CODEOWNERS

```diff
@@ -1,8 +1,8 @@
-# Require review from golang-compiler team for changes in any file. This keeps us in the loop on
-# auto-merge PRs. The review bot is also an owner so that it can still trigger auto-merge for sync
-# PRs on its own. We may remove this rule once auto-merges are routine.
-* @microsoft/golang-compiler @microsoft-golang-review-bot
+# Require review from golang-compiler team for changes in all files.
 
-# Automatically request review from golang-compiler team for changes in the Microsoft-specific
-# files. This takes precedence over earlier rules in the file.
-/eng/ @microsoft/golang-compiler
+* @microsoft/golang-compiler
+
+# Don't assign a code owner to automatically updated files to allow GitHub apps to use the
+# auto-merge flow without human intervention.
+/go
+/manifest.json
```

Comment:
I think practically speaking, we need to have all of `/src/microsoft/` in here. Take a look at https://github.com/microsoft/go-images/pull/389/files for example.

Or perhaps `/src/microsoft/1.*/` + `/src/microsoft/versions.json`.

Reply:
```suggestion
/manifest.json
/src/microsoft/
```